pkgs/development/python-modules/can/default.nix

```diff
@@ -1,29 +1,41 @@
 { lib
 , buildPythonPackage
-, fetchPypi
+, fetchFromGitHub
+, pythonOlder
+, isPy33
+, isPy34
 , wrapt
+, typing
 , pyserial
 , nose
 , mock
+, hypothesis
+, future
 , pytest
 , pytest-timeout }:
 
 buildPythonPackage rec {
   pname = "python-can";
-  version = "3.0.0";
+  version = "3.1.0";
 
-  src = fetchPypi {
-    inherit pname version;
-    sha256 = "0d2ddb3b663af51b11a4c7fb7a577c63302a831986239f82bb6af65efc065b07";
+  # PyPI tarball is missing some tests and is missing __init__.py in test
+  # directory causing the tests to fail. See:
+  # https://github.com/hardbyte/python-can/issues/518
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "hardbyte";
+    rev = "v${version}";
+    sha256 = "01lfsh7drm4qvv909x9i0vnhskdh27mcb5xa86sv9m3zfpq8cjis";
   };
 
-  propagatedBuildInputs = [ wrapt pyserial ];
-  checkInputs = [ nose mock pytest pytest-timeout ];
+  disable = isPy33 || isPy34;
```

Comment:
```suggestion
```
Those are no longer available in Nixpkgs.

Reply:
Thanks, made the correction!